apps/frontend/src/pages/[type]/[id].vue

```diff
@@ -1090,7 +1060,18 @@ const licenseIdDisplay = computed(() => {
     return id;
   }
 });
-
+function setSelectedGameVersion(version) {
+  userSelectedGameVersion.value = version;
+  if (import.meta.client) {
```

Comment:
Rather than actually calling localStorage, it might be best to use `useLocalStorage` composable provided by vueuse, which handles the `import.meta.client` stuff for you automatically.

Also, make sure that it correctly falls back to undefined/not set if the local storage game version isn't actually supported by the project.